sherpa/utils/tests/test_utils.py

```diff
@@ -388,3 +389,14 @@ def test_filter_bins_unordered():
     assert len(flags) == len(expected)
     for got, exp in zip(flags, expected):
         assert got == exp
+
+def test_paralle_map_funcs():
+    for arg in range(1, 5):
+        func = [numpy.sum]
+        funcs = arg * func
+        datas = [numpy.arange(1, 2+2*i) for i in range(arg)]
+        result = []
+        for func, data in zip(funcs, datas):
+            result.extend(numpy.asarray(list(map(func, data))))
+        assert_equal(result, utils.parallel_map_funcs(funcs, datas, arg))
+
```

Comment:
I think we need a "real" test, too, that uses all the sherpa machinery with setting up data, defining a model through the ui etc.

I know this is marked as WIP, so I understand it's just not here yet, but I'm just saying we'll need it eventually.